laws/shared/src/main/scala/cats/laws/discipline/Arbitrary.scala

```diff
@@ -44,6 +44,16 @@ object arbitrary {
       A.arbitrary.map(a => Eval.later(a)),
       A.arbitrary.map(a => Eval.always(a))))
 
+  import cats.data.{Streaming, StreamingT}
+
+  implicit def streamingArbitrary[A](implicit A: Arbitrary[A]): Arbitrary[Streaming[A]] =
+    Arbitrary(Gen.listOf(A.arbitrary).map(Streaming.fromList))
```

Comment:
It would be nice to generate some lazy values too, but with the way scalacheck implements `() => A`, I don't know that we could do anything useful here.